libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -651,6 +651,19 @@ impl NodeAPI for Greenlight {
         .map(|credentials| NodeCredentials::Greenlight { credentials }))
     }
 
+    async fn set_node_config(&self, close_to_address: Option<String>) -> NodeResult<()> {
+        let mut client = self.get_client().await?;
+        if let Some(close_to_addr) = close_to_address {
+            client
+                .configure(gl_client::pb::GlConfig { close_to_addr })
```

Comment:
Is this where validation happens?

If a user were to make a typo or give an invalid address, is this where it would fail?

Reply:
If not, would it make sense to add our own
```rust
ensure_sdk!(
    bitcoin::util::address::Address::from_str(&close_to_addr).is_ok(),
    // SdkError
);
```

Reply:
I don't think the address is validated in the gl-plugin. Probably a good idea to add validation.

Reply:
Ok I checked, there is validation: https://github.com/Blockstream/greenlight/blob/6ab8f3205639ddf611f99e71691f7515fc0d24e6/libs/gl-plugin/src/node/mod.rs#L465-L509